stresstest-aws/docker-compose.tf

```diff
@@ -61,6 +63,11 @@ x-quorum-def:
         --minerthreads 1 \
         --syncmode full \
 %{endif~}
+%{if var.consensus == "clique"~}
+        --mine \
+        --minerthreads 1 \
+        --syncmode full \
+%{endif~}
```

Comment:
similar solution like recommendation for extraData, we can have `local.consensus_geth_args` map.
Simplify the template with lesser if conditions

Reply:
i tried to use a map to lookup but docker-compose was failing as the code was not getting formatted correctly(left indents were not getting applied though i tried sapce/tabs).